tests/snapshots/TestCLICommands_Valid_log_level_in_env_should_be_priortized_over_config.stderr.golden

```diff
@@ -1,3 +1,4 @@
+DEBU Log level set to DEBU file=/dev/stderr
 DEBU Found ENV var ATMOS_LOGS_LEVEL=Debug
 DEBU Found ENV var ATMOS_VERSION_CHECK_ENABLED=false
 DEBU processStoreConfig atmosConfig.StoresConfig=map[]
```

Comment:
What concerns me is that we emit found after we emit set. Something seems off. For example, if we said "ignoring ENV" I would understand. Found implies we will use it.

Reply:
Working on this :)

Reply:
Done